tools/util.py

```diff
@@ -374,14 +376,12 @@ def mkdtemp():
 
 
 class DenoTestCase(unittest.TestCase):
-    @property
-    def build_dir(self):
-        args = test_args()
-        return args.build_dir
+    @classmethod
+    def setUpClass(cls):
+        args = parse_test_args()
 
-    @property
-    def deno_exe(self):
-        return os.path.join(self.build_dir, "deno" + executable_suffix)
+        cls.build_dir = args.build_dir
+        cls.deno_exe = os.path.join(args.build_dir, "deno" + executable_suffix)
```

Comment:
Perhaps we can use `args.executable or os.path.join(args.build_dir, "deno" + executable_suffix)`